### PR TITLE
Guidelines to encourage collegiate atmosphere

### PR DESCRIPTION
**Pull request by andycroll**
My thoughts from analysing the 'so far' videos.

### CODE REVIEW

**Comment by nodunayo** [comment]
👍 

Very much agree with Andy's suggestions.

File: docs/game/cheat_sheet.md [hunk 1]
```diff
@@ -60,13 +65,14 @@ Deviation:
 Tactics
 -------
 
-* Speaking in a slow, measured way is beneficial. Just don't slow down to a stop.
+* Speaking in a measured way is beneficial, although be careful not to slow down too much - try and maintain your normal speaking voice and variation in tone.
```
Comment (AJFaraday) [inline docs/game/cheat_sheet.md:68]: We may need to rethink this line. Some of the most entertaining runs in the 'real world' are when someone dons a character, or intentionally changes their voice in some way. (Case in point, Paul Merton recently did a whole round in an impression of Ringo Starr). 

While I realise that I'm not working with professional entertainers, I don't want to discourage that kind of behaviour.

File: docs/game/cheat_sheet.md [hunk 2]
```diff
@@ -4,10 +4,14 @@ Panellist Cheat Sheet
...
+Relax, take it easy and have fun. The panel should have a generous, collegiate
+but competitive air.
 
-It's a little competitive, but ultimately we're all pulling together to make an
-enjoyable, entertaining and informative show. Winning isn't everything.
+Ultimately we're all pulling together to make an enjoyable, entertaining and 
+informative show. Think performance, rather than deathmatch.
+
+Trying too hard to win can lead to one voice dominating and variety is the spice
+of life. And this game. _Winning points isn't the point, or how you "win"._
```
Comment (AJFaraday) [inline docs/game/cheat_sheet.md:14]: You're starting a sentence with 'And' here. 

`of life, and`

</nitpick>

File: docs/game/cheat_sheet.md [hunk 3]
```diff
@@ -4,10 +4,14 @@ Panellist Cheat Sheet
 Golden Rule!
 ------------
 
-Relax, take it easy and have fun.
+Relax, take it easy and have fun. The panel should have a generous, collegiate
```
Comment (AJFaraday) [inline docs/game/cheat_sheet.md:7]: :+1: :+1: :+1:

**Comment by AJFaraday** [comment]
I've just jotted down a couple of thoughts on these changes.

It's also reminded me that I need to write some guidelines on 'upping the ante'.

File: docs/game/cheat_sheet.md (hunk 1 above)
Comment (nodunayo) [inline docs/game/cheat_sheet.md:68]: Maybe you want to point out that your voice shouldn't come across as distorted on account of you trying to speak slowly.

File: docs/game/cheat_sheet.md (hunk 2 above)
Comment (nodunayo) [inline docs/game/cheat_sheet.md:14]: I mean — I'm with you 100% on the grammatical point.

However, it's for (comic) effect so it works...but it's very much Andy's voice and not yours so...

**Pull request merged**